ConstraintLayoutExamples/motionlayout/src/main/java/com/google/androidstudio/motionlayoutexample/histogramdemo/HistogramActivity.kt

```diff
@@ -0,0 +1,147 @@
+package com.google.androidstudio.motionlayoutexample.histogramdemo
+
+import android.os.Bundle
+import android.view.View
+import android.view.ViewTreeObserver.OnGlobalLayoutListener
+import androidx.annotation.Nullable
+import androidx.appcompat.app.AppCompatActivity
+import androidx.constraintlayout.motion.widget.MotionLayout
+import androidx.constraintlayout.motion.widget.MotionLayout.TransitionListener
+import androidx.core.content.ContextCompat
+import com.google.androidstudio.motionlayoutexample.R
+import kotlinx.android.synthetic.main.histogram_layout.*
```

Comment:
Now we have ViewBiding (https://developer.android.com/topic/libraries/view-binding), how about using it instead of the Kotlin extension?

Reply:
Hmm unfortunately current 4.0 canary 1 seems to be having an issue with data binding (gradle failure). I've filed a bug. Maybe my env is bit messed up :( 

Could I submit a separate pull request? I'm ok waiting and having everything in a single PR as well. Whatever makes it easier to review :)

Reply:
Sure, a separate pull request sounds good. Please note that my suggestion was to use ViewBinding instead of Data Binding :)